versions.tf

```diff
@@ -1,3 +1,3 @@
 terraform {
-  required_version = ">= 1.3"
+  required_version = ">= 1.8"
```

Comment:
Change to rollback, we already have this constrain in the .tofu file for tofu only